Shared/GridController.h

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ *  Copyright 2012-2025 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+
+#ifndef GRIDCONTROLLER_H
+#define GRIDCONTROLLER_H
+
+// DSA headers
+#include "AbstractTool.h"
+
+// Qt headers
+#include <QString>
+#include <QVariantMap>
```

Comment:
You can probably forward declare this one. QtContainerFwd should have all the forwards declares to make that easy. No need to copy paste out of that file, just include it here. Then you can include `<QVariantMap>` over in the .cpp file.